lib/spack/spack/cmd/install.py

```diff
@@ -61,7 +62,11 @@ def setup_parser(subparser):
     subparser.add_argument(
         '--run-tests', action='store_true', dest='run_tests',
         help="Run tests during installation of a package.")
-
+    subparser.add_argument(
+        '-p', '--install-policy', action='store', dest="install_policy",
```

Comment:
Sounds better. What about `--cached-binary` as `binary` again is a little bit too generic?